csrankings-l.csv

```diff
@@ -392,6 +394,7 @@ Li Weigang 0001,Universidade de Brasília,http://cic.unb.br/~weigang,Sw_i7yQAAAA
 Li Xiao 0001,Michigan State University,http://www.cse.msu.edu/~lxiao,NOSCHOLARPAGE
 Li Xiao 0004,University of Aberdeen,https://www.abdn.ac.uk/people/xiao.li,dZLlkYQAAAAJ
 Li Xiong 0001,Emory University,https://cs.emory.edu/~lxiong,jJ8BLgsAAAAJ
+Li Yan,Univ. of Maryland - Baltimore County,https://www.csee.umbc.edu/people/faculty/li-yan/,NOSCHOLARPAGE

```

Comment:
This is a disambiguation DBLP entry. Either it needs to be augmented with the appropriate suffix or you need to follow the directions on communicating with dblp@dblp.org.

Reply:
removed